packages/core-data/src/hooks/use-entity-records.ts

```diff
@@ -97,8 +108,35 @@ export default function useEntityRecords< RecordType >(
 		[ kind, name, queryAsString, options.enabled ]
 	);
 
+	const { totalItems, totalPages } = useSelect(
+		( select ) => {
+			if ( ! options.enabled ) {
+				return {
+					// Avoiding returning a new reference on every execution.
```

Comment:
Is this about the value of props? I'd think we would need to have a `const`  and return the same object.

Reply:
this comment has been copy pasted I think, I'll remove it.